index.bs

```diff
@@ -2160,7 +2225,19 @@ Constructors</h4>
 			2. Set the {{[[rendering thread state]]}} for
 				<var>c</var> to <code>"suspended"</code>.
 
-			3. Construct an {{AudioDestinationNode}} with its
+			3. Determine the {{[[render quantum size]]}} for this
+				 {{OfflineAudioContext}}, based on the value of
+				 the {{OfflineAudioContextOptions/renderSizeHint}}:
+
+				 1. If it has the default value of <code>"default"</code> or
```

Comment:
I am a bit confused about the step (1). Does this mean that if `[render quantum size]` is equal to `default` or `hardware` then we use 128?

Reply:
This is for `OfflineAudioContext`, not `AudioContext`.